modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipFormBase.tsx

```diff
@@ -126,13 +146,26 @@ export function ObjectRelationshipFormBase({
 	const [objectDefinitions, setObjectDefinitions] = useState<
 		Partial<ObjectDefinition>[]
 	>([]);
-	const [query, setQuery] = useState<string>('');
 
-	const types = useMemo(() => {
-		return OBJECT_RELATIONSHIP_TYPES.filter((relationshipType) =>
-			objectRelationshipTypes?.includes(relationshipType.value)
-		);
-	}, [objectRelationshipTypes]);
+	const [currentObjectDefinition, setCurrentObjectDefinition] = useState<
```

Comment:
@rebsilva Could you sort those states by alphabetical order? Also, you don't need to leave the blank lines between them because the order of execution would not matter for them.